### DISCUSSION TITLE
command

### DISCUSSION THREAD

**Discussion by sidd0798** [community/community#33569] [Other Feature Feedback, Questions, & Ideas]
when i writing the push command for   second file in in my directory it is showing  [rejected]........error: failed to push some refs to 'https://github.com/sidd0798/git_practice_assignmnet.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Comment by Matteo-it** [comment]
Try to do this:
- git pull
- git push

**Comment by airtower-luna** [comment]
The error message tells you what the problem is: There are changes in your GitHub repository that aren't in your local repository. If the push went ahead, those would be lost. What to do depends on whether you want to keep those changes:

* If you want to keep the remote changes, too, either *merge* the remote branch into your local one (this will add a merge commit), or *rebase* your local branch on the remote one.
* If you want to  the remote-only changes, you can force-push. That means you tell Git to push even if it means losing data, so you should be really sure it's what you want before you do it. :warning: 

You can use `git log` and `git diff` to compare the branches. Make sure to fetch first so you have the current remote data.
